src/rust/grapl-observe/src/metric_reporter.rs

```diff
@@ -36,6 +36,17 @@ pub struct MetricReporter<W: std::io::Write> {
     service_name: String,
 }
 
+impl MetricReporter<Stdout> {
+    pub fn new(service_name: &str) -> Self {
+        MetricReporter {
+            service_name: service_name.to_string(),
+            buffer: String::new(),
+            out: WriterWrapper::new(stdout()),
+            utc_now: Utc::now,
+        }
+    }
+}
+
 /**
 some followup TODOs:
     - add tags to the public functions (not needed right now)
```

Comment:
What's this? If it's something we need to action maybe it needs a github issue and the note made more clear? Otherwise can we remove the note?

Reply:
That TODO isn't from my diff, I'd ask @wimax-grapl